src/prefect/environments/storage/gcs.py

```diff
@@ -39,6 +39,10 @@ def __init__(self, bucket: str, key: str = None, project: str = None) -> None:
 
         super().__init__()
 
+    @property
+    def labels(self) -> List[str]:
+        return ["gcs-storage"]
```

Comment:
Nit pick, consider "gcs-flow-storage" (and similar in other places) to maximize specificity and prevent potential overlap with future features that we may distinct with similar labels

Reply:
Yeah I'm on board with `-flow-storage` because `gcs-storage` seems like something that a user could tag their flow with if it interfaces with their GCS storage